content/projects/gsoc/2020/project-ideas/github-checks.adoc

```diff
@@ -0,0 +1,104 @@
+---
+layout: gsocprojectidea
+title: "GitHub Checks API for Jenkins Plugins"
+goal: "Create a new plugin API so that plugins can publish GitHub checks status messages"
+category: Plugins
+year: 2020
+status: published
+sig: platform
```

Comment:
Fine with me, but not sure whether it was intended. CC @MarkEWaite

Reply:
Ah, I just copied that from another document. What should be used here?